src/main/java/io/opencensus/integration/jdbc/Observability.java

```diff
@@ -14,12 +14,12 @@
 
 package io.opencensus.integration.jdbc;
 
+import com.sun.istack.internal.Nullable;
```

Comment:
s/com.sun.istack.internal.Nullable/javax.annations.Nullable

Reply:
Yes we did not have the jsr305 linked. Added that as well.